udata_gouvfr/tests.py

```diff
@@ -445,7 +443,7 @@ def test_oembed_town_territory_api_get(self, api):
             assert cgi.escape(territory.url) in html
             assert 'alt="{name}"'.format(name=organization.name) in html
             assert md(territory.description, source_tooltip=True) in html
-            assert 'Download from localhost' in html
+            assert 'Download from local.test' in html
```

Comment:
Hum, those changes should be on the current master :/ (I will submit the PR unless you do)

Reply:
Please do :-)